keylime.conf

```diff
@@ -198,23 +198,25 @@ registrar_private_key_pw = default
 # Database URL Configuration
 # See this document https://keylime-docs.readthedocs.io/en/latest/installation.html#database-support
 # for instructions on using different database configurations.
-# Database configuration via drivername, username, password, etc is deprecated
-# and superseded by database_url, when database_url is set, other database
-# fields will be ignored.
+# There are two options for the specification of the database connection configuration.
+# 1) A tuple of parameters (e.g., database_drivername, database_username, database_password)
+# 2) A single string in database_url. This, when set, takes precedence over the previous option.
 # An example of database_url value for using sqlite:
 #   sqlite:////var/lib/keylime/cv_data.sqlite
 # An example of database_url value for using mysql:
 #   mysql+pymysql://keylime:keylime@keylime_db:[port]/verifier?charset=utf8
-# If database_url is not set, the deprecated default is "sqlite" and will be
+# If database_url is not set, the default is "sqlite", using method 1, and will be
 # located at "/var/lib/keylime/cv_data.sqlite".
 database_url =
-drivername = sqlite
-username = ''
-password = ''
-host = ''
-port = ''
-database = cv_data.sqlite
-query = ''
+database_drivername = sqlite
+database_username = ''
+database_password = ''
+database_host = ''
+database_port = ''
+database_name = cv_data.sqlite
+database_query = ''
+# Limits for DB connection pool size in sqlalchemy (https://docs.sqlalchemy.org/en/14/core/pooling.html#api-documentation-available-pool-implementations)
+database_pool_sz_ovfl=5,10
```

Comment:
While these may be better names for these config items, I am wondering what will happen on a system that is upgraded from the old config to the new config where these items are renamed. An rpm for example treats config files differently not overwriting the user's config, thus leading to a malfunctioning system.

Reply:
I agree. I would like to see the old config values still be valid but deprecated and printing an info message. Then we can completely remove them at a later point.

Reply:
fair point, will make the necessary changes

Reply:
I just realized that I pushed the new code with a bug. I am out of the office today, will push a fix soon